diff --git a/app/src/behavior.c b/app/src/behavior.c
--- a/app/src/behavior.c
+++ b/app/src/behavior.c
@@ -119,7 +119,7 @@ static int validate_hid_usage(uint16_t usage_page, uint16_t usage_id) {
     switch (usage_page) {
     case HID_USAGE_KEY:
         if (usage_id == 0 ||
-            (usage_id > ZMK_HID_KEYBOARD_NKRO_MAX_USAGE &&
+            (usage_id > ZMK_HID_KEYBOARD_MAX_USAGE &&
              usage_id < HID_USAGE_KEY_KEYBOARD_LEFTCONTROL) ||
             usage_id > HID_USAGE_KEY_KEYBOARD_RIGHT_GUI) {
             return -EINVAL;
